internal/commands/up_infra.go

```diff
@@ -130,7 +145,195 @@ func (o *UpInfraOptions) Validate() error {
 }
 
 func (o *UpInfraOptions) Run() error {
+	if o.Explain {
+		tmpl := `# Change to the dir
+cd {{.EnvDir}}
+
+# Generate Backend.tf
+cat << EOF > backend.tf
+provider "aws" {
```

Comment:
It would be best to use backend config template as it is actually generated, and not recreate it manually. Can we do that?

Reply:
No